src/shared/custom-rpc/getPayloadId.ts

```diff
@@ -0,0 +1,12 @@
+let base = 1;
+const typedArray = new Uint16Array(1);
+
+/**
+ * returns a unique random value always larger than the previous one
+ */
+export function getPayloadId(): number {
+  if (base >= 2 ** 15 - 1) {
```

Comment:
maybe it would be better to make it a named constant, like "max int16" or smth like that